docs/4.3/installation.md

```diff
@@ -37,43 +9,72 @@ The following examples install the 64-bit version of Teleport binaries, but
 Release](https://gravitational.com/teleport/download/) page for the most
 up-to-date information.
 
-**Debian**
-```bash
-$ curl https://get.gravitational.com/teleport_{{ teleport.version }}_amd64.deb.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport_{{ teleport.version }}_amd64.deb
-$ sha256sum teleport_{{ teleport.version }}_amd64.deb
-# Verify that the checksums match
-$ dpkg -i teleport_{{ teleport.version }}_amd64.deb
-$ which teleport
-/usr/local/bin/teleport
-```
-
-**Tarball**
-```bash
-$ curl https://get.gravitational.com/teleport-v{{ teleport.version }}linux-amd64-bin.tar.gz.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-$ shasum -a 256 teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-# Verify that the checksums match
-$ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
-$ cd teleport
-$ ./install
-$ which teleport
-/usr/local/bin/teleport
-```
-
-**RPM**
-```bash
-$ curl https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm.sha256
-# <checksum> <filename>
-$ curl -O https://get.gravitational.com/teleport-{{ teleport.version }}-1.x86_64.rpm
-$ sha256sum teleport-{{ teleport.version }}-1.x86_64.rpm
-# Verify that the checksums match
-$ rpm -i teleport-{{ teleport.version }}-1.x86_64.rpm
-$ which teleport
-/usr/local/bin/teleport
-```
+=== "Tarball"
+
+    ```bash
+    $ curl https://get.gravitational.com/teleport-v{{ teleport.version }}linux-amd64-bin.tar.gz.sha256
+    # <checksum> <filename>
+    $ curl -O https://get.gravitational.com/teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    $ shasum -a 256 teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    # Verify that the checksums match
+    $ tar -xzf teleport-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    $ cd teleport
+    $ ./install
+    $ which teleport
+    /usr/local/bin/teleport
+    ```
+
+=== "Debian"
```

Comment:
Should probably be `DEB` (as there are various other distros that aren't Debian but use the `.deb` package format)

Reply:
Updated